tests/test_authorizer.py

```diff
@@ -340,6 +341,60 @@ def test_refresh(self):
         self.assertEqual(set(["*"]), authorizer.scopes)
         self.assertTrue(authorizer.is_valid())
 
+    # def test_refresh_with__2fa(self):
```

Comment:
I don't think this example is necessary. If you want to add some information in the documentation about running tests with 2FA that'd be great, but not necessary for this PR.

Reply:
Ok. I included it just in case you wanted to verify that the feature works, since the other tests don't really prove it.